simapp/proposal.go

```diff
@@ -0,0 +1,13 @@
+package simapp
+
+import (
+	"github.com/cosmos/cosmos-sdk/x/gov"
+	"github.com/cosmos/cosmos-sdk/x/gov/types"
+)
+
+var _ gov.MsgSubmitProposal = MsgGovSubmitProposal{}
+
+func (m MsgGovSubmitProposal) GetContent() types.Content {
+	return m.Proposal.GetTextProposal()
```

Comment:
How does this work exactly? The content is an interface, right? Can you show an example of at least two concrete `content` types just so we can see what that logic would look like? How do you know what concrete type to return?